task2/InterfaceManager.py

```diff
@@ -0,0 +1,109 @@
+import json
+import logging
+from dataclasses import dataclass, field
+from typing import List
+
+import marshmallow as mm
+import requests
+from dataclasses_json import DataClassJsonMixin, config
+from marshmallow import fields, EXCLUDE
+
+# Logger
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")
+
+file_handler = logging.FileHandler('output_folder/interfaces.log', mode='w')
+file_handler.setLevel(logging.INFO)
+file_handler.setFormatter(formatter)
+
+logger.addHandler(file_handler)
```

Comment:
@patrikprovaznik put this to some util module that will take as an input (log_name: str, folder_path_name: str) that will be predefined and return the logger which you will use here